.github/pull_request_template.md

```diff
@@ -31,8 +31,9 @@ https://raw.githubusercontent.com/ceph/ceph/master/SubmittingPatches.rst
 
 -->
 ## Checklist
-- [ ] References tracker ticket
-- [ ] Updates documentation if necessary
+- [ ] References [tracker ticket](https://tracker.ceph.com/issues/new)
+  - [ ] References tracker ticket to [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new) and [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new) (if necessary)
```

Comment:
I think the original "References tracker ticket" referred to adding a `Fixes: https://tracker.ceph.com/issues/12345` tag to the commit message of one of the commits so that the ticket is recorded in the git history.  It sounds like what you are after with the dashboard and/or orchestrator tickets is less formal -- just having them filed so they make it to the appropriate team's list?

Reply:
Thanks @idryomov ! Yes, what we wanted here is that devels link the deps/changes required from deployment and UI management perspective. Any suggestion here? Changing "references" with...?

Reply:
how about "Affects Dashboard and/or Orchestrator, open tracker ticket (if necessary)"?

Reply:
Like it! Thanks, @neha-ojha !